desktop/install/linux-install.md

```diff
@@ -268,6 +270,58 @@ Docker Desktop for Linux runs a Virtual Machine (VM) for the following reasons:
 
     As such, we have adjusted the default memory available to the VM in DD4L. You can tweak this setting to your specific needs by using the **Memory** slider within the **Settings** > **Resources** tab of Docker Desktop.
 
+## File sharing
+
+Docker Desktop for Linux uses [virtiofs](https://virtio-fs.gitlab.io/) as the
+default (and currently only) mechanism to enable file sharing between the host
+and Docker Desktop VM. In order not to require elevated privileges, without
```

Comment:
```suggestion
and Docker Desktop VM. To remove the need for elevated privileges, without
```

Reply:
I changed it to "To avoid requiring". "remove" sounded to me like we had required elevated privileges previously, which is not the case. The current choice of implementation was more about avoiding an alternative that would, even though the choice comes with its own complications.